packages/components/pie-toast-provider/src/defs.ts

```diff
@@ -1,3 +1,54 @@
-// TODO - please remove the eslint disable comment below when you add props to this interface
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface ToastProviderProps {}
+import { type ToastProps } from '@justeattakeaway/pie-toast';
+
+import { type ComponentDefaultProps } from '@justeattakeaway/pie-webc-core';
+
+export const PRIORITY_ORDER: { [x: string]: number } = {
+    'error-actionable': 1,
+    error: 2,
+    'warning-actionable': 3,
+    'success-actionable': 4,
+    'info-actionable': 5,
+    'neutral-actionable': 6,
+    warning: 7,
+    success: 8,
+    info: 9,
+    neutral: 10,
+};
```

Comment:
I just tried this out locally and it seems to give us better intellisense and avoids using `string` which feels too generic. I was considering asking you to add a string union of the possible priorities too but actually if we use this `as const` we can instead get a string union from `keyof typeof PRIORITY_ORDER` if needed...
```typescript
export const PRIORITY_ORDER = {
    'error-actionable': 1,
    error: 2,
    'warning-actionable': 3,
    'success-actionable': 4,
    'info-actionable': 5,
    'neutral-actionable': 6,
    warning: 7,
    success: 8,
    info: 9,
    neutral: 10,
} as const;
```

Reply:
will be addressed in the docs PR 🤝